privmsg.go

```diff
@@ -255,12 +256,14 @@ func (robo *Robot) learn(ctx context.Context, log *slog.Logger, ch *channel.Chan
 		log.DebugContext(ctx, "no learn tag")
 		return
 	}
+	start := time.Now()
```

Comment:
Start time should move down a line so it measures only the latency of the database-y operation.